### PR TITLE
UI contacts

### PR DESCRIPTION
**Pull request by popkinj**
## Enhancements
- Wallet information
- Table of contacts
- Button for adding new contact
- Page component layout

## Adjustments
- Moved the `add contact` component into a modal

**Recommending a Squash Merge.**

### CODE REVIEW

**Comment by popkinj** [comment]
Definitely need to squash merge this sucker.

File: services/grip/frontend/.env.test [hunk 1]
```diff
@@ -0,0 +1 @@
+VITE_TRACTION_ENDPOINT=https://traction-api-test.apps.silver.devops.gov.bc.ca:5100
```
Comment (loneil) [inline services/grip/frontend/.env.test:1]: We'll have to adjust how the environmental settings for Traction URLs get into the app at deployment time when we figure out our hosting and Helm/OCP plans
https://apps.nrs.gov.bc.ca/int/jira/browse/TRACTION-314

File: services/grip/frontend/.env.development [hunk 2]
```diff
@@ -0,0 +1 @@
+TRACTION_ENDPOINT=https://traction-api-dev.apps.silver.devops.gov.bc.ca:5100
```
Comment (loneil) [inline services/grip/frontend/.env.development:1]: There's a few of these "No newline at end of file" warnings

File: services/grip/frontend/src/components/Contacts.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<script setup lang="ts">
+import { inject, ref } from "vue";
+import Fieldset from "primevue/fieldset";
+import DataTable from "primevue/datatable";
+import Column from "primevue/column";
+import ProgressSpinner from "primevue/progressspinner";
+import axios from "axios";
+import Button from "primevue/button";
+import Dialog from "primevue/dialog";
+import CreateContact from "./CreateContact.vue";
```
Comment (loneil) [inline services/grip/frontend/src/components/Contacts.vue:10]: Not important but would be nice to group the primevue imports in order together

File: services/grip/frontend/.env.test (hunk 1 above)
Comment (popkinj) [reply services/grip/frontend/.env.test:1]: Yeah. I'm starting to think we should either configure NGinx to proxy the API or maybe spin up a little Node server to serve the assets and proxy.

File: services/grip/frontend/src/components/Contacts.vue (hunk 3 above)
Comment (popkinj) [reply services/grip/frontend/src/components/Contacts.vue:10]: Agreed... that would be much cleaner. I'll check to see if Rollup can handle that.

**Pull request merged**